client/apps/shipping-label/view-wrapper-label.js

```diff
@@ -162,6 +163,17 @@ export class ShippingLabelViewWrapper extends Component {
 		this.props.openTrackingFlow( orderId, siteId );
 	};
 
+	updateOrderDetailScreen = () => {
+		if ( window.jQuery ) {
+			window.jQuery.get( window.location.href, function( result ) {
+				const parsedResult = window.jQuery( result );
+				const updatedOrderDetails = parsedResult.find( '#order_data' );
+				const updatedOrderNotes = parsedResult.find( '#woocommerce-order-notes' );
+				window.jQuery( '#woocommerce-order-notes' ).html( updatedOrderNotes.html() );
+				window.jQuery( '#order_data' ).html( updatedOrderDetails.html() );
```

Comment:
Instead of creating a new API endpoint just to get the updated notes, we perform a GET request to the current page (order's detail page) to get its latest state, and just update the page sections we're interested in.

Reply:
Order notes works great but using the method for updating `#order_data` has problems. All the dynamic features of that card do not work. The order datepicker, customer auto completer and edit buttons on the address no longer function correctly when replacing the html like this. I suggest we stick with the line just to update the order status:
```
jQuery('#order_status').val('wc-completed').trigger('change');
```